### PR TITLE
Devex at IW

### PR DESCRIPTION
**Pull request by PaulLucas-IW**

### CODE REVIEW

**Comment by PaulLucas-IW** [comment]
**NB:** Don't merge until I've cleared the project descriptions with Gaffers and/or Clare for publication.

File: _posts/2018-04-17-devx-at-iw.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+The projects and challenges that we tackle are many and varied and that variety is really at the heart of our DevEx.
+
+We push ourselves to learn different domains, experience different ways of working and understand a wide range of technical stacks.  We work across the stack and across the SDLC.  Engineering to us doesn't just mean being able to be a great developer, it also means helping out with the infrastructure-as-code, building the deployment pipelines and integrating automated operational monitoring.
+
+In real-terms that means working on projects like building and running cloud-based services for a large betting &amp; gaming provider or helping the NHS introduce a more agile culture, while adding modern tooling like docker and re-engineering critical systems to save money and lives.
+
+We're working with a large retailer to build out a new loyalty platform for their customers and instill a true DevOps engineering culture.  The new microservice architecture is cloud-hosted on Kubernetes and fully automated - we regularly do more than 100 production deployments a day.
```
Comment (jakehall-IW) [inline _posts/2018-04-17-devx-at-iw.md]: SP: instil

File: _posts/2018-04-17-devx-at-iw.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+At a more personal level we try to make our engineering experience better by letting people choose their laptop and which dev tools they want to licence.  For those working in our offices it means little things like providing decent monitors and loads of wall space for information radiators and whiteboards (so many whiteboards!).
+
+![Events this week at Infinity Works]({{ "/images/event-board.jpg" }}){: .inline-image}
+
+Aside from the actual project work we do many other things including a weekly yoga session in our Leeds office, running clubs, lunches, community events and cross-team catch ups.  It's really important that people feel like they're part of Infinity Works.
+
+When you get stuck, behind you is a community of really great engineers who can offer help and guidance and with whom you can share problems and ideas.  We put a lot of effort into fostering that community spirit though talks (both internally and externally), meet-ups and collaborative tooling like Slack.
+
+Our company vision is "Best for Customer, Best for Collegue" and getting both of those in balance is integral to improving our DevEx.
```
Comment (jakehall-IW) [inline _posts/2018-04-17-devx-at-iw.md]: SP: Colleague

File: _posts/2018-04-17-devx-at-iw.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+## The C-Word
+
+When talking to people who are looking to join us we often come across some pre-conceived notions about what the DevEx at a consultancy looks like.  These are often gained from working with or alongside the large, more traditional consultancies.
+
+Infinity Works offers a different experience.  For a start we are much smaller and that gives us the freedom to do the right things without enormous amounts of red tape and bureauracy.
```
Comment (jakehall-IW) [inline _posts/2018-04-17-devx-at-iw.md]: SP: bureaucracy

File: _posts/2018-04-17-devx-at-iw.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+layout: post
+title: DevEx at Infinity Works
+author: paul_lucas
+image: laptop_desk.jpg
+---
+
+We attended [QCon in London](https://qconlondon.com) last month (which as an aside, I highly recommend if you've not been before) and one of the day's tracks that caught my attention was looking at "DevEx" or "Developer Experience" - the daily life of developers/engineers and how that might be improved through better tooling, process and skills.  It got me thinking - what is the "DevEx" for engineers at Infinity Works and what are the things we can do to nuture it?
+
+> "DevEx is not just about the tools, but the daily experience at work"
+> - QCon Presentation
```
Comment (jakehall-IW) [inline _posts/2018-04-17-devx-at-iw.md]: Possible to be more specific about the presentation and link to it maybe?

File: _posts/2018-04-17-devx-at-iw.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+So what _does_ it look like to be part of Infinity Works?  What is our day-to-day "DevEx"?
+
+## "There's _always_ something going on, Danny!"
+
+Engineering at Infinity Works is first and foremost about solving real-world problems.  We work as trusted delivery partners for our clients helping them to deliver their projects, figuring out solutions to hard technical problems and bringing a strong culture of Agile and DevOps along with us.
+
+The projects and challenges that we tackle are many and varied and that variety is really at the heart of our DevEx.
+
+We push ourselves to learn different domains, experience different ways of working and understand a wide range of technical stacks.  We work across the stack and across the SDLC.  Engineering to us doesn't just mean being able to be a great developer, it also means helping out with the infrastructure-as-code, building the deployment pipelines and integrating automated operational monitoring.
```
Comment (jakehall-IW) [inline _posts/2018-04-17-devx-at-iw.md]: Is it worth linking out to SDLC on wiki, incase people don't immediately understand the initialism?

File: _posts/2018-04-17-devx-at-iw.md (hunk 4 above)
Comment (PaulLucas-IW) [reply _posts/2018-04-17-devx-at-iw.md]: Yeah, I've left that deliberately vague as it's from one of our competitors :)

File: _posts/2018-04-17-devx-at-iw.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: DevEx at Infinity Works
+author: paul_lucas
+image: laptop_desk.jpg
+---
+
+We attended [QCon in London](https://qconlondon.com) last month (which as an aside, I highly recommend if you've not been before) and one of the day's tracks that caught my attention was looking at "DevEx" or "Developer Experience" - the daily life of developers/engineers and how that might be improved through better tooling, process and skills.  It got me thinking - what is the "DevEx" for engineers at Infinity Works and what are the things we can do to nuture it?
```
[the remark this answers is not in the document]
Comment (PaulLucas-IW) [reply _posts/2018-04-17-devx-at-iw.md]: Fixed

File: _posts/2018-04-17-devx-at-iw.md (hunk 5 above)
Comment (PaulLucas-IW) [reply _posts/2018-04-17-devx-at-iw.md]: Changed wording

File: _posts/2018-04-17-devx-at-iw.md (hunk 1 above)
Comment (PaulLucas-IW) [reply _posts/2018-04-17-devx-at-iw.md]: Fixed

File: _posts/2018-04-17-devx-at-iw.md (hunk 2 above)
Comment (PaulLucas-IW) [reply _posts/2018-04-17-devx-at-iw.md]: Fixed

File: _posts/2018-04-17-devx-at-iw.md (hunk 3 above)
Comment (PaulLucas-IW) [reply _posts/2018-04-17-devx-at-iw.md]: Fixed

**Comment by PaulLucas-IW** [comment]
Oi @neildunlop, I'm merging this tomorrow even if you haven't looked at it :)

**Comment by neildunlop** [comment]
I have looked.. I did a whole other branch! Did you not like any of It? I
might cry!!! Zsolt commented on some of my words

**Comment by PaulLucas-IW** [comment]
So you did.  I wasn't expecting the pull-request-caption...  will review and merge!

**Comment by neildunlop** [comment]
Nice!

**Pull request merged**